src/Jackett.Common/Definitions/pixelados.yml

```diff
@@ -97,13 +100,36 @@ search:
   fields:
     category:
       selector: category_id
-    title:
+    types:
+      selector: types
+    megapack:
+      selector: types:contains(MEGAPACK)
+      optional: true
+    mejorada:
+      selector: types:contains(Mejorada)
```

Comment:
Please double check: the standard unit3d API 6.1.0 returns `type` 
Is `types` a customisation by the pixelados team or is this a typo?